generated_docker_darwin.go

```diff
@@ -195,6 +217,15 @@ type (
 		// Since: generic-worker 5.3.0
 		Features FeatureFlags `json:"features,omitempty"`
 
+		// Image to use for the task.  Images can be specified as an image tag as used by a docker registry, or as an object declaring type and name/namespace
+		//
+		// One of:
+		//   * DockerImageName
+		//   * NamedDockerImage
+		//   * IndexedDockerImage
+		//   * DockerImageArtifact
+		Image json.RawMessage `json:"image,omitempty"`
```

Comment:
The reason this is generated as `json.RawMessage` is because `additionalProperties: false` is not specified.